include/v4l2_camera/rate_bound_status.hpp

```diff
@@ -38,11 +40,11 @@ namespace custom_diagnostic_tasks
  */
 struct RateBoundStatusParam
 {
-  RateBoundStatusParam(const double min_freq, const double max_freq)
+  RateBoundStatusParam(const double min_freq, const std::optional<double> max_freq = std::nullopt)
```

Comment:
`Constructors that are callable with a single argument must be marked explicit to avoid unintentional implicit conversions (fix available)clang-tidy[google-explicit-constructor](https://clang.llvm.org/extra/clang-tidy/checks/google/explicit-constructor.html)`
```suggestion
  explicit RateBoundStatusParam(const double min_freq, const std::optional<double> max_freq = std::nullopt)
```